Manito/Manito/Presentation/Scene/LoginScene/ViewController/LoginViewController.swift

```diff
@@ -0,0 +1,132 @@
+    }
+    
+    private func bindOutputToViewModel(_ output: LoginViewModel.Output?) {
+        guard let output else { return }
+        
+        output.isNewLogin
+            .receive(on: DispatchQueue.main)
+            .sink(receiveValue: { [weak self] result in
+                switch result {
+                case .success(let isNewLogin):
+                    if isNewLogin {
+                        self?.presentCreateNicknameViewController()
+                    } else {
+                        self?.presentMainViewController()
+                    }
+                case .failure(let error):
+                    switch error {
+                    case .failedToLogin: self?.makeAlertWhenLoginError(error: error.localizedDescription)
+                    case .failedCredential: self?.makeAlertWhenCredentialError(error: error.localizedDescription)
+                    case .failedToken: self?.makeAlertWhenTokenError(error: error.localizedDescription)
+                    case .failedTokenToString: self?.makeAlertWhenTokenToStringError(error: error.localizedDescription)
+                    }
+                }
+            })
+            .store(in: &self.cancellable)
+    }
+}
+
+// MARK: - Helper
+extension LoginViewController {
+    private func presentCreateNicknameViewController() {
+        let repository = SettingRepositoryImpl()
+        let service = NicknameService(repository: repository)
+        let viewModel = NicknameViewModel(nicknameService: service)
+        let viewController = CreateNicknameViewController(viewModel: viewModel)
+        self.navigationController?.pushViewController(viewController, animated: true)
+    }
+    
+    private func presentMainViewController() {
+        let navigationController = UINavigationController(rootViewController: MainViewController())
+        navigationController.modalPresentationStyle = .fullScreen
+        navigationController.modalTransitionStyle = .crossDissolve
+        self.present(navigationController, animated: true)
+    }
+    
+    private func makeAlertWhenLoginError(error: String) {
+        self.makeAlert(title: TextLiteral.Common.Error.title.localized(),
+                       message: error,
+                       okAction: nil)
+    }
+    
+    private func makeAlertWhenCredentialError(error: String) {
+        self.makeAlert(title: TextLiteral.Sign.Error.credential.localized(),
+                       message: error,
+                       okAction: nil)
+    }
+    
+    private func makeAlertWhenTokenError(error: String) {
+        self.makeAlert(title: TextLiteral.Sign.Error.token.localized(),
+                       message: error,
+                       okAction: nil)
+    }
+    
+    private func makeAlertWhenTokenToStringError(error: String) {
+        self.makeAlert(title: TextLiteral.Sign.Error.tokenToString.localized(),
+                       message: error,
+                       okAction: nil)
+    }
```

Comment:
p3;

```suggestion
    private func makeAlertWhenLoginError(error: String) {
        self.makeAlert(title: TextLiteral.Common.Error.title.localized(),
                       message: error)
    }
    
    private func makeAlertWhenCredentialError(error: String) {
        self.makeAlert(title: TextLiteral.Sign.Error.credential.localized(),
                       message: error)
    }
    
    private func makeAlertWhenTokenError(error: String) {
        self.makeAlert(title: TextLiteral.Sign.Error.token.localized(),
                       message: error)
    }
    
    private func makeAlertWhenTokenToStringError(error: String) {
        self.makeAlert(title: TextLiteral.Sign.Error.tokenToString.localized(),
                       message: error)
    }
```

action에 들어갈 동작이 없다면 굳이 적지 않아도 됩니다!

Reply:
[9008946](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/580/commits/90089460743b27a22035cda1f7a699b66a292ef1)
수정했어요